src/PointwiseFunctions/MathFunctions/Gaussian.hpp

```diff
@@ -62,21 +71,38 @@ class Gaussian : public MathFunction<1> {
 
   double operator()(const double& x) const noexcept override;
   DataVector operator()(const DataVector& x) const noexcept override;
+  using MathFunction<1, Fr>::operator();
 
   double first_deriv(const double& x) const noexcept override;
   DataVector first_deriv(const DataVector& x) const noexcept override;
+  using MathFunction<1, Fr>::first_deriv;
 
   double second_deriv(const double& x) const noexcept override;
   DataVector second_deriv(const DataVector& x) const noexcept override;
+  using MathFunction<1, Fr>::second_deriv;
 
   double third_deriv(const double& x) const noexcept override;
   DataVector third_deriv(const DataVector& x) const noexcept override;
+  using MathFunction<1, Fr>::third_deriv;
+
+  double amplitude() const noexcept { return amplitude_; }
```

Comment:
@geoffrey4444 Note to self: in next fixup commit, remove these accessor functions that I forgot to remove

Reply:
ping?